sphinx_gallery 0.1.13 error

With sphinx_gallery 0.1.13 from pypi I get the following error when running `make html-noplot`:

```
Exception occurred:
  File "/tmp/tutorials/custom_directives.py", line 91, in run
    intro = sphinx_gallery.gen_rst.extract_intro(abs_fname)
AttributeError: module 'sphinx_gallery.gen_rst' has no attribute 'extract_intro'
The full traceback has been saved in /tmp/sphinx-err-5xzryf86.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:48: html-noplot] Error 1

```

--- a/custom_directives.py
+++ b/custom_directives.py
@@ -88,7 +88,8 @@ class GalleryItemDirective(Directive):
             if 'intro' in self.options:
                 intro = self.options['intro'][:195] + '...'
             else:
-                intro = sphinx_gallery.gen_rst.extract_intro(abs_fname)
+                _, blocks = sphinx_gallery.gen_rst.split_code_and_text_blocks(abs_fname)
+                intro, _ = sphinx_gallery.gen_rst.extract_intro_and_title(abs_fname, blocks[0][1])
 
             thumbnail_rst = sphinx_gallery.backreferences._thumbnail_div(
                 dirname, basename, intro)

